attiny402_firmware/src/serial.cpp

```diff
@@ -1,8 +1,6 @@
 #include "serial.h"
-#include <WS2812.h>
 
-WS2812 LED(1);
-cRGB val;
+uint8_t led_mode = 0;
```

Comment:
Wouldn't an enum be more appropriate? @Cogno-Marco

Reply:
`enum class` though

Reply:
also in `led.cpp` because of the `extern uint8_t led_mode`

Reply:
can you do a quick commit because I didn't quite understand @_@